WooCommerce/Classes/ViewRelated/Orders/Order Creation/OrderForm.swift

```diff
@@ -282,55 +282,59 @@ private struct ProductsSection: View {
-                    switch capturePermissionStatus {
-                    case .notPermitted:
-                        logPermissionStatus(status: .notPermitted)
-                        self.showPermissionsSheet = true
-                    case .notDetermined:
-                        logPermissionStatus(status: .notDetermined)
-                        viewModel.requestCameraAccess(onCompletion: { isPermissionGranted in
-                            if isPermissionGranted {
-                                showAddProductViaSKUScanner = true
-                                logPermissionStatus(status: .permitted)
-                            }
-                        })
-                    case .permitted:
-                        showAddProductViaSKUScanner = true
-                        logPermissionStatus(status: .permitted)
+                HStack {
+                    Button(OrderForm.Localization.addProducts) {
+                        showAddProduct.toggle()
                     }
-                }
-                .accessibilityIdentifier(OrderForm.Accessibility.addProductViaSKUScannerButtonIdentifier)
-                .buttonStyle(PlusButtonStyle())
-                .sheet(isPresented: $showAddProductViaSKUScanner, onDismiss: {
-                    scroll.scrollTo(addProductViaSKUScannerButton)
-                }, content: {
-                    ProductSKUInputScannerView(onBarcodeScanned: { detectedBarcode in
-                        viewModel.addScannedProductToOrder(barcode: detectedBarcode, onCompletion: { _ in
-                            showAddProductViaSKUScanner.toggle()
+                    .id(addProductButton)
+                    .accessibilityIdentifier(OrderForm.Accessibility.addProductButtonIdentifier)
+                    .buttonStyle(PlusButtonStyle())
+                    .sheet(isPresented: $showAddProduct, onDismiss: {
+                        scroll.scrollTo(addProductButton)
+                    }, content: {
+                        ProductSelectorNavigationView(
+                            configuration: ProductSelectorView.Configuration.addProductToOrder(),
+                            isPresented: $showAddProduct,
+                            viewModel: viewModel.productSelectorViewModel)
+                        .onDisappear {
+                            viewModel.productSelectorViewModel.clearSearchAndFilters()
+                            navigationButtonID = UUID()
+                        }
+                    })
+
+                    Button(action: {
+                        let capturePermissionStatus = viewModel.capturePermissionStatus
+                        switch capturePermissionStatus {
+                        case .notPermitted:
+                            logPermissionStatus(status: .notPermitted)
+                            self.showPermissionsSheet = true
+                        case .notDetermined:
+                            logPermissionStatus(status: .notDetermined)
```

Comment:
Do we need these log statements here? Seems to me like debug code that we can remove from `trunk`

Reply:
I'd say is worth to keep them as makes easier for support folks to debug any issue a merchant may have with the camera, it was pretty common to see tickets where merchants couldn't upload images/make pictures because their permissions were disabled.